internal-ui/src/components/DashboardComponent.tsx

```diff
@@ -30,111 +83,174 @@ export const SelectFilters = () => {
   const rangeString = startDate && endDate
     ? `${startDate.toLocaleDateString()} TO ${endDate.toLocaleDateString()}`
     : startDate
-    ? `${startDate.toLocaleDateString()} TO (end date not selected)`
-    : endDate
-    ? `(start date not selected) TO ${endDate.toLocaleDateString()}`
-    : 'No date selected';
+      ? `${startDate.toLocaleDateString()} TO (end date not selected)`
+      : endDate
+        ? `(start date not selected) TO ${endDate.toLocaleDateString()}`
```

Comment:
toLocaleDateString will give the local time, but the database uses only one time. Probably should lock it to EST.

Also, you can probably delete this range segment. The user can see the calendar, and doesn't need a print out :) -- maybe keep it for debugging the date-time problems I mentioned.